recipes/boutdata/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "boutdata" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8f0148f00c66c75f7bac6392af73e35676b65395e19496c2c10cd6f586097ac5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.0
+    - pip
+  run:
+    - python >=3.0
+    - sympy
+    - numpy
+    - matplotlib-base
+    - scipy
+    - bunch
+    - boututils
+
+test:
+  imports:
+    - boutdata
+  requires:
+    - pytest
+  source_files:
+    - test
+  commands:
+    - pytest
```

Comment:
```suggestion
```

Reply:
Why do you suggest removing the `pytest` test @marcelotrevisani?

Reply:
@marcelotrevisani sorry I didn't understand your suggestion. Is the change needed before merging? Thanks!